### PR TITLE
first part completed

### PR DESCRIPTION
**Pull request by FatemaGirnary**

### CODE REVIEW

File: ETL_Mini_Project_TRadadiya_ZLiang_YWong_FGirnary.ipynb [hunk 1]
```diff
@@ -234,7 +234,8 @@
     }
    ],
    "source": [
-    "# Get a brief summary of the crowdfunding_info DataFrame.\n"
+    "# Get a brief summary of the crowdfunding_info DataFrame.\n",
+    "crowdfunding_info_df.info(15)"
```
Comment (TruptiRadadiya) [inline ETL_Mini_Project_TRadadiya_ZLiang_YWong_FGirnary.ipynb]: Please remove 15 from the code as just "crowdfunding_info_df.info()" is valid

File: ETL_Mini_Project_TRadadiya_ZLiang_YWong_FGirnary.ipynb [hunk 2]
```diff
@@ -426,79 +428,225 @@
...
+     "execution_count": 10,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "# Assign the category and subcategory values to category and subcategory columns.\n"
+    "# Assign the category and subcategory values to category and subcategory columns.\n",
+    "crowdfunding_info_df[['category', 'subcategory']] = crowdfunding_info_df['category & sub-category'].str.split('/', expand=True)\n",
+    "crowdfunding_info_df"
```
Comment (TruptiRadadiya) [inline ETL_Mini_Project_TRadadiya_ZLiang_YWong_FGirnary.ipynb]: please print just first five rows. use crowdfunding_info_df.head()

File: ETL_Mini_Project_TRadadiya_ZLiang_YWong_FGirnary.ipynb [hunk 3]
```diff
@@ -555,21 +703,24 @@
...
+   "execution_count": 18,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Create a category DataFrame with the category_id array as the category_id and categories list as the category name.\n",
+    "category_df = pd.DataFrame({'category_id': cat_ids,\n",
+    "                            'category': categories})\n",
     "\n",
     "# Create a category DataFrame with the subcategory_id array as the subcategory_id and subcategories list as the subcategory name. \n"
    ]
```
Comment (TruptiRadadiya) [inline ETL_Mini_Project_TRadadiya_ZLiang_YWong_FGirnary.ipynb:581]: Add a code to create subcategory dataframe as below
subcategory_df = pd.DataFrame({'subcategory_id': scat_ids,
                            'subcategory': subcategories})

**Review by TruptiRadadiya — COMMENTED** [review]
Kindly make requested changes and push for review

File: ETL_Mini_Project_TRadadiya_ZLiang_YWong_FGirnary.ipynb (hunk 1 above)
Comment (TruptiRadadiya) [reply ETL_Mini_Project_TRadadiya_ZLiang_YWong_FGirnary.ipynb]: Just the last change. please resolve above. everything else seems good.

**Review by TruptiRadadiya — APPROVED** [review]

**Review by Jean-Liang — APPROVED** [review]

**Pull request merged**